packages/@aws-cdk/aws-logs/lib/query-definition.ts

```diff
@@ -0,0 +1,232 @@
+import { Resource } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnQueryDefinition } from '.';
+
+
+/**
+ * Properties for a QueryString
+ */
+export interface QueryStringProps {
+  /**
+  * Retrieves the specified fields from log events for display.
+  *
+  * @default no fields in QueryString
+  */
+  readonly fields?: string;
+
+  /**
+  * Extracts data from a log field and creates one or more ephemeral fields that you can process further in the query.
+  *
+  * @default no parse in QueryString
+  */
+  readonly parse?: string;
+
+  /**
+  * Filters the results of a query that's based on one or more conditions.
+  *
+  * @default no filter in QueryString
+  */
+  readonly filter?: string;
+
+  /**
+  * Uses log field values to calculate aggregate statistics.
+  *
+  * @default no stats in QueryString
+  */
+  readonly stats?: string;
+
+  /**
+  * Sorts the retrieved log events.
+  *
+  * @default no sort in QueryString
+  */
+  readonly sort?: string;
+
+  /**
+  * Specifies the number of log events returned by the query.
+  *
+  * @default no limit in QueryString
+  */
+  readonly limit?: Number;
+
+  /**
+  * Specifies which fields to display in the query results.
+  *
+  * @default no display in QueryString
+  */
+  readonly display?: string;
+}
+
+/**
+ * Define a QueryString
+ */
+export class QueryString {
+  /**
+  * Retrieves the specified fields from log events for display.
+  */
+  private fields?: string;
+
+  /**
+  * Extracts data from a log field and creates one or more ephemeral fields that you can process further in the query.
+  */
+  private parse?: string;
+
+  /**
+  * Filters the results of a query that's based on one or more conditions.
+  */
```

Comment:
This shouldn't have the docs duplicated from the corresponding member of `QueryStringProps`, I think these docs can be removed entirely because the class member is private. Same applies to the other members.

Reply:
Removed.